src/lib/sw.js

```diff
@@ -161,7 +163,7 @@ self.addEventListener("fetch", (event) => {
 
   const p = Promise.resolve().then(async () => {
     // First, check if there's actually something in the cache already.
-    const cacheKey = workbox.precaching.getCacheKeyForURL(
+    const cacheKey = workboxPrecaching.getCacheKeyForURL(
```

Comment:
https://developers.google.com/web/tools/workbox/guides/migrations/migrate-from-v4#reading_precached_responses

In v5, you can simplify these multiple statements to

```
import {matchPrecache} from 'workbox-precaching';

// ...

const cached = await matchPrecache(u.pathname + "/index.html");
```